test/build/50_dockerfile.bats

```diff
@@ -1,38 +1,44 @@
 load ../common
 
+# FIXME: this function is temporary; remove when references to ch-grow are gone.
+setup () {
+    if [[ $CH_BUILDER = ch-grow ]]; then
+       CH_BUILDER=ch-image
+    fi
+}
```

Comment:
It still works with `CH_BUILDER = ch=grow` though doesn't it?

Reply:
Ah, yes but if you examine this file you see that there is a lot of ` if [[ $CH_BUILDER == ch-image ]]; [...]`. It is much simpler here to just add this temporary setup function rather than edit all of these if statements to:

```
if [[ $CH_BUILDER == ch-image || $CH_BUILDER == ch-grow ]]; [...]
```

Reply:
Aha, OK. I was thinking of the scripts rather than the test suite.